docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -69,7 +69,7 @@ We primarily support development on Linux and Unix-based systems like Ubuntu and
 | Prerequisite                                                                                  | Version | Notes                                                                                       |
 | --------------------------------------------------------------------------------------------- | ------- | ------------------------------------------------------------------------------------------- |
 | [Node.js](http://nodejs.org)                                                                  | `18.x`  | We use the "Active LTS" version, See [LTS Schedule](https://nodejs.org/en/about/releases/). |
-| [pnpm](https://pnpm.io/installation)                                                          | `8.x`   | -                                                                                            |
+| [pnpm](https://pnpm.io/installation)                                                          | `8.6.x`   | -                                                                                            |
```

Comment:
Undo this change. 

Instead, add the error text and troubleshooting info here or in the FAQ later.

The notes bit in the table should read like so:

```markdown
If you see the error: 'Your ... <add the full text>' See [troubleshooting](link to the troubleshooting).
```

The troubleshooting FAQ should read like so:

```markdown
pnpm updates the lock file. If do not have the latest of version of `pnpm` then simply update your `pnpm` by either following the installation [instructions here](https://pnpm.io/installation) OR run `npm install -g pnpm@latest`
```

Reply:
For GitPod users separate changes can be done by adding a pre/post setup steps in the config to achieve same results.